routers/api/v1/api.go

```diff
@@ -688,6 +688,11 @@ func RegisterRoutes(m *macaron.Macaron) {
 							m.Post("/start", reqToken(), repo.StartIssueStopwatch)
 							m.Post("/stop", reqToken(), repo.StopIssueStopwatch)
 						})
+						m.Group("/subscriptions", func() {
+							m.Get("", bind(api.User{}), repo.GetIssueWatchers)
```

Comment:
I think watchers is the term used for repos and subscribers for issues/prs, but I could be wrong.
```suggestion
							m.Get("", bind(api.User{}), repo.GetIssueSubscribers)
```

Reply:
the db talbe is also caled `issue_watch` but for external things it should called issueSubscribe to show difference betwen repo and issue watch and subscribe ... Changed it :)